.github/workflows/FissionESLintFormat.yml

```diff
@@ -24,30 +24,14 @@ jobs:
         npm install
     - name: Linter
       id: linter-validation
+      if: ${{ always() }}
```

Comment:
Is the if always really necessary? We don't use it in our other workflows, would like to keep them the same if possible.

Reply:
It's necessary for it to continue on error but still report errors properly. I think we want to get both ESLint and prettier errors, but when we use continue-on-error it always says the step passed. With `if always` it will still run both but the steps will fail if ESLint and prettier fail.

Reply:
Understood, as long as there is a specific reason for it.